src/chrome/content/rules/Conviva.xml

```diff
@@ -3,26 +3,20 @@
 
 		- (www.)
```

Comment:
You should update this comment.
`(www.)` seems working now but there are still some broken ones.

Mismatch on:
static.conviva.com
lp.conviva.com